src/main/java/net/neoforged/moddevgradle/internal/ModDevPlugin.java

```diff
@@ -259,6 +260,15 @@ public void apply(Project project) {
             minecraftClassesArtifact = createArtifacts.map(task -> project.files(task.getCompiledArtifact()));
         }
 
+        var supplyDevLogin = project.provider(() -> extension.getRuns().stream().anyMatch(model -> model.getDevLogin().get()));
+
+        // Create a configuration to resolve DevLogin
```

Comment:
Can you inject it the same way we inject DevLaunch? That way it will also work in neodev.

Reply:
What do you mean by that? In the aftereval? @shartte Suggested doing it this way but I can do it there instead. I believe he wanted it added this way to make debugging work?

Reply:
Please check my commit.